test/include/test_helper/test_helper.h

```diff
@@ -25,6 +25,7 @@ struct TestQueryConfig {
 class TestHelper {
 public:
     inline static std::string E2E_TEST_FILES_DIRECTORY = "test/test_files";
+    inline static std::string E2E_IMPORT_TEST_DIR = "dataset/";
```

Comment:
Let's not conflate the two import types into this single variable as it hides the intent.
Instead, let the existing code remain as it is and use `E2E_OVERRIDE_IMPORT_DIR` or equivalent, with a default empty value. That way it is clear that there is an override of the default import type.